controllers/controllers.js

```diff
@@ -6,10 +6,15 @@ exports.getTopics = (req, res) => {
   });
 };
 
-exports.getArticles = (req, res) => {
-  selectArticles().then(articles => {
+exports.getArticles = (req, res,next) => {
+  const topic = req.query.topic
+  const sort_by = req.query.sort_by
+  const order = req.query.order
```

Comment:
A tidying up refactor could be to destructure these properties off the `req.query` object - turning 3 lines into one!